src/Console/Commands/MvcContractsUpdate.php

```diff
@@ -31,7 +31,7 @@ class MvcContractsUpdate extends Command
      */
     public function handle(DocumentSignInterface $documentSign)
     {
-        $contracts = Contract::whereNotNull('external_id')->whereNull('signed_at')->get();
+        $contracts = Contract::whereNotNull('external_id')->whereNull('signers')->get();
```

Comment:
this is incorrect, beacuse it might be that one signer hasn't signed yet and another did, than we still have to update the other signer.

What is the contract status before STATUS_SIGNED? I would retrieve those contracts: where('status', STATUS_...)

Reply:
i guess STATUS_SENT is the correct one but check yourself?